### DISCUSSION TITLE
Child devices pictures not shown in Rack page

### DISCUSSION THREAD

**Discussion by mauriciopeclatlopes** [netbox-community/netbox#7763] [Q&A]
Netbox 3.0.9
I have a shelf with two populated bays. The child devices were created based on a device type containing its front picture.

Why the child picture is not shown in the rack?
 
![Captura de Tela 2021-11-05 às 15 51 08](https://user-images.githubusercontent.com/20050659/140563406-96f713d0-e57a-4677-99b6-9ffbdf8f02f2.png)

**Comment by candlerb** [comment]
> Why the child picture is not shown in the rack?

Because Netbox has no way to specify how device bays are laid out in a parent device.

For example: a 6U blade server may have 12 child devices.  Are these 12 vertical blades, each taking 6U vertically?  Or six horizontal blades, each taking 1U vertically and half width?  Or something else?

It would require some sort of grid overlaid on the parent device, to define the physical space provided by each device bay.
